### DISCUSSION TITLE
APP UTN FRSR

### DISCUSSION THREAD

**Discussion by NadiaCruz1** [CodeSystem2022/Grupo_Programaci-nUTN-TercerSemestre#1] [Q&A]
Para comenzar con la creación de la app que nos pidieron desde UTN FRSR, necesitamos algunas ideas sobre que debería llevar esta.

¿Qué tarareas debería realizar?
¿Cuáles van a ser sus características principales?
Analizar todas las necesidades que el usuario podría tener.
Deberíamos intentar responder estas preguntas y ver que es lo que es lo que el usuario necesita.

**Comment by CamilaGonzalez97** [comment]
Algunas ideas que podríamos integrar  son:
Acceso a información relevante:
1. Calendario académico
2. Noticias sobre y de la facultad 
3. Horarios de clases
4. fechas de exámenes
5. Notificaciones y recordatorios sobre eventos (tratando de que lleguen solo a quienes quieran participar en el mismo)
6. Notificaciones sobre cambios de horarios o suspensiones de actividades.

**Reply by NahuelAlaniz** [reply]
Podríamos realizar  acceso a materiales de aprendizaje como notas de clase(pueden ser apuntes que suban los mismos alumnos y quieran compartir ), power, videos, libros . También  generar un sitio/ comunidad donde los alumnos- docentes y no docentes puedan comunicarse(para preguntar sobre materias y clases, asi como disponibilidad de aulas ).
 Se podría generar una sección de tutorías, un espacio donde se pueda guardar el progreso del estudiante y un seguimiento sobre la asistencia a clase (que notifique en cuanto este llegando al limite de inasistencias )

**Comment by ayelenlscabral** [comment]
hola equipo!
podríamos agregar a nuestra aplicación elementos interactivos como botones y menús, implementar la funcionalidad para acceder a la información de la UTN, como horarios de clase, noticias, eventos ,entre otros. esto podría incluir la integración de una API o el acceso de una base de datos.
agregar funcionalidades adicionales como notificaciones push, integración con redes sociales, entre otras, según sea necesario.
en mi opinión como ejemplo básico creo que hay muchos otros factores y consideraciones a tener en cuenta al crear la aplicación.

**Comment by NicolasBogado** [comment]
Como alumnos, creo que se necesita estar actualizado con las noticias, y las actividades. La app creo que se debe centrar principalmente en eso, comunicación constante alumno-docente. Y me sumo a la idea del resto del equipo.
